api/mapping/services.py

```diff
@@ -133,7 +133,7 @@ def process_scan_report(scan_report_id):
                         is_patient_id=False,
                         is_date_event=False,
                         is_ignore=False,
-                        pass_from_source=False,
+                        pass_from_source=True,
```

Comment:
@simon-tarr, could you set pass_from_source default equals True in the models instead of here. It will make sure that we got this rule independent from where we create the object.